eodag/resources/providers.yml

```
@@ -1383,7 +1387,6 @@
     base_uri: 'https://mundiwebservices.com/dp'
     extract: true
     auth_error_code: 401
```

```suggestion
    auth_error_code: 401
    bucket_path_level: 0
```
restores https://github.com/CS-SI/eodag/pull/524